src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
+html {
+  height: 100%;
+}
+
 body {
   margin: 0;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  position: relative;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes {
+    position: absolute;
+    left: calc(50% - 1.5vmin);
+    top: calc(50% - 20vmin);
+    display: block;
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    animation: nev 3600s infinite steps(60);
+    transform-origin: bottom center;
+  }
+
+  &__minutes--speed-up {
+    animation-duration: 600s;
+  }
+
+  &__seconds {
+    position: absolute;
+    top: calc(50% - 38vmin);
+    left: calc(50% - 1vmin);
+    display: block;
+    width: 1.5vmin;
+    height: 38vmin;
+    background-color: #2c8000;
+    transform: rotate(1deg);
+    animation: nev 60s infinite;
+    transform-origin: bottom center;
+    animation-timing-function: linear;
+  }
```

Comment:
The `transform: rotate(1deg);` here might be a bit misleading. This rotates the element 1 degree from its original position. However, since this is a clock hand, it might be more intuitive to start it at 0 degrees, and let the animation handle the rotation. This is not a critical issue, but it's something to consider for code readability.

Reply:
thanks